### PR TITLE
Go entry project

### PR DESCRIPTION
**Pull request by tokyohmachine**
### Description

 I created a new pull request with the updated file path and changes. Now, I'm waiting for the repository maintainers to review and hopefully accept my new contributions.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by caupolicandiaz — APPROVED** [review]
@tokyohmachine I've gone ahead and made some changes that were missing from the previous review. I also removed the php edits from the other PR. Please review the changes for future reference. I'm going to pass this along for a second review.

File: content/go/concepts/strings/terms/join/join.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+This example results in the following output:
+
+```shell
+Hello, World
+```
+
+## Codebyte Example
+
+In the code below several strings are concatenated into a single string, separated by spaces, using the `Join()` function.
```
Comment (CBID2) [inline content/go/concepts/strings/terms/join/join.md:57]: [Typo]
```suggestion
In the code below, several strings are concatenated into a single string, separated by spaces, using the `Join()` function.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Looks great @tokyohmachine. Just found a typo.

**Review by SSwiniarski — APPROVED** [review]
@tokyohmachine I committed @CBID2's suggestion because otherwise this looks good to merge.

**Comment by SSwiniarski** [comment]
@tokyohmachine congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/go/strings/join

image

**Pull request merged**